tests/utils/test_json.py

```diff
@@ -0,0 +1,25 @@
+"""test json."""
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from supervisor.utils.json import write_json_file
+
+
+def test_file_permissions():
+    """Test file permissions."""
+    with TemporaryDirectory() as temp_dir:
```

Comment:
Pytest has a `tmp_path` fixture.
https://docs.pytest.org/en/stable/tmpdir.html

Reply:
Correct, but we are not using that anywhere else?